sklearn/model_selection/tests/test_validation.py

```diff
@@ -56,7 +56,7 @@
 from sklearn.datasets import make_classification
 from sklearn.datasets import make_multilabel_classification
 
-from test_split import MockClassifier
+from .test_split import MockClassifier
```

Comment:
According to http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines, tests should use absolute imports: 

```python

from sklearn.model_selection.tests.test_split import MockClassifier
```

I guess it wasn't done this way originally because of the missing `__init__.py` ...